.vsconfig

```diff
@@ -0,0 +1,47 @@
+{
+  "version": "1.0",
+  "components": [
```

Comment:
As mentioned in the previous comment, we should just included what's needed but set `includeRecommended = true`  in the settings:
```
          "Microsoft.VisualStudio.Workload.ManagedDesktop"
          "Microsoft.VisualStudio.Workload.Universal"
```